tests/functional/aws-node-sdk/test/object/completeMPU.js

```diff
@@ -138,5 +140,71 @@ describe('Complete MPU', () => {
                 _completeMpuAndCheckVid(uploadId, eTag, undefined, done);
             });
         });
+
+        describe('with tags set on initiation', () => {
+            const tagKey = 'keywithtags';
+
+            taggingTests.forEach(test => {
+                it(test.it, done => {
+                    const key = encodeURIComponent(test.tag.key);
+                    const value = encodeURIComponent(test.tag.value);
+                    const tagging = `${key}=${value}`;
+
+                    async.waterfall([
+                        next => s3.createMultipartUpload({
+                            Bucket: bucket,
+                            Key: tagKey,
+                            Tagging: tagging,
+                        }, (err, data) => {
+                            if (test.error) {
+                                assert(err, 'Expected err but found none');
```

Comment:
I tend to not include asserts like this that would be checked anyway by the following asserts. Meaning, if this would fail, so would the next check so we can skip it.
Like if we want `a` to equal 5, we do not need to check that its a number first just that its 5.